spanner/src/create_instance_with_processing_units.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/spanner/README.md
+ */
+
+namespace Google\Cloud\Samples\Spanner;
+
+// [START spanner_create_instance_with_processing_units]
+use Google\Cloud\Spanner\SpannerClient;
+
+/**
+ * Creates an instance.
+ * Example:
+ * ```
+ * create_instance_with_processing_units($instanceId);
+ * ```
+ *
+ * @param string $instanceId The Spanner instance ID.
+ */
+function create_instance_with_processing_units($instanceId)
```

Comment:
We should make it standard to use typehints / return types for all samples.
```suggestion
function create_instance_with_processing_units(string $instanceId): void
```